specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimapis.json

```diff
@@ -3435,6 +4314,271 @@
+          "description": "Comment text."
+        },
+        "createdDate": {
+          "type":"string",
+          "format": "date-time",
+          "description": "Date and time when the comment was created."
+        },
+        "userId": {
+          "type": "string",
+          "description": "A resource identifier for the user who left the comment."
+        }
+      },
+      "required": [
+        "text",
+        "userId"
+      ],
+      "description": "Issue Comment contract Properties."
+    },
+    "IssueAttachmentCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IssueAttachmentContract"
+          },
+          "description": "Issue Attachment values."
+        },
+        "count": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Total record count number across all pages."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any."
+        }
+      },
+      "description": "Paged Issue Attachment list representation."
+    },
+    "IssueAttachmentContract": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/IssueAttachmentContractProperties",
+          "description": "Properties of the Issue Attachment."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./apimanagement.json#/definitions/Resource"
+        }
+      ],
+      "description": "Issue Attachment Contract details."
+    },
+    "IssueAttachmentContractProperties": {
+      "properties": {
+        "title": {
+          "type": "string",
+          "description": "Short description of the attachment."
+        },
+        "contentFormat": {
+          "type": "string",
+          "description": "Content format.",
+          "enum": [
+            "link"
+          ],
+          "x-ms-enum": {
+            "name": "AttachmentContentFormat",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "link",
+                "description": "The Content is a link."
+              }
+            ]
+          }
+        },
+        "content": {
+          "type": "string",
+          "description": "An HTTP link to the attachment."
+        }
+      },
+      "description": "Issue Attachment contract Properties."
+    },
+    "IssueAttachmentPutContract": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/IssueAttachmentContractProperties",
+          "description": "Properties of the Issue Attachment for the Put operation."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./apimanagement.json#/definitions/Resource"
+        }
+      ],
+      "description": "Issue Attachment Contract details."
+    },
+    "IssueAttachmentPutContractProperties": {
```

Comment:
not used anywhere - thanks to the example you provided for the put operation, this was caught by our tooling 😃 Pretty sure you wanted to use this schema for `IssueAttachmentPutContract`'s `properties`. (My naming comment regarding using "Put" in the name of this type of course also applies for this type)